### PR TITLE
updated old default version of minio

### PR DESCRIPTION
**Pull request by nicholas-goodwin**
## [Ticket Link #714](https://app.zenhub.com/workspaces/mdot-615b97c1a5fde400126174f8/issues/urbanos-public/internal/714)

Remove if not applicable

## Description

What was changed

## Reminders

- [x] Did you up the relevant chart version numbers? (If appropriate)
  - [x] If you up a chart version within urban-os, have you also upped the urban-os chart version itself?
  - [x] If charts within the urban-os chart (andi, etc) have been updated, have you run `helm dependency update` in /charts/urban-os and commited the Chart.lock file?
- [x] If chart values added, were default values provided in the chart? (Will `helm template . -f values.yaml` pass?)
~~- [ ] Do you have git hooks installed? (See README.md to install)~~
~~- [ ] If references to external charts were added:~~
  ~~- [ ] Was the github release action updated to `helm update {new_thing}` it's dependencies?~~
  ~~- [ ] Was the deploy repo `-u` flag updated to `helm update {new_thing}` to ensure it's not left out of deployments?~~

### CODE REVIEW

File: charts/persistence/README.md [hunk 1]
```diff
@@ -1,19 +1,31 @@
...
+| https://trinodb.github.io/charts/ | trino | 0.8.0 |
+
 ## Values
 
 | Key | Type | Default | Description |
 |-----|------|---------|-------------|
+| global.buckets.hiveStorageBucket | string | `"presto-hive-storage"` |  |
 | global.objectStore.accessKey | string | `"example-key"` |  |
-| global.objectStore.hiveStorageBucket | string | `"presto-hive-storage"` |  |
+| global.objectStore.accessSecret | string | `"example-secret"` |  |
```
Comment (jfolk2015) [inline charts/persistence/README.md:25]: Does this need to be set somewhere?

**Comment by bmitchinson** [comment]
@nicholas-goodwin I'm not gonna give approval since I co-authored, maybe @jfolk2015 or @RyanRConaway could do a review.

Could you revert the KDP chart changes though? Persistence will replace it in time, so let's not make further maintenance to KDP

File: charts/persistence/values.yaml [hunk 2]
```diff
@@ -69,3 +69,8 @@ trino:
       hive.s3.path-style-access=true
       hive.s3.endpoint=http://minio:80
       hive.s3.ssl.enabled=false
+
+# minio operator
+operator:
+  enabled: false
```
Comment (bmitchinson) [inline charts/persistence/values.yaml:75]: I think enabled false by default is the right call here 👍 
Let's keep it that way for the external charts as opt in.

File: charts/persistence/README.md (hunk 1 above)
Comment (nicholas-goodwin) [reply charts/persistence/README.md:25]: If I remember correctly, this is a placeholder for secrets to populate it

Comment (bmitchinson) [reply charts/persistence/README.md:25]: There are values described in the base chart values too to make it clear this needs to be set 👍 
https://github.com/UrbanOS-Public/charts/blob/f432ea7e674ddf5c5ada8c1832662826ea4c99b9/charts/persistence/values.yaml#L7

Docs were just never updated

Comment (jfolk2015) [reply charts/persistence/README.md:25]: That's what I would expect as well, from the kubernetes secrets right? Is setting these values on the clusters part of this card?

Comment (nicholas-goodwin) [reply charts/persistence/README.md:25]: setting those values are not part of this card

**Review by jfolk2015 — APPROVED** [review]

**Pull request merged**